platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,231 @@
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.command;
+
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.apache.karaf.bundle.core.BundleService;
+import org.apache.karaf.features.Dependency;
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.features.FeaturesService;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.admin.application.service.ApplicationServiceException;
+import org.codice.ddf.security.common.Security;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceReference;
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.security.Subject;
+import ddf.security.SubjectUtils;
+
+import static org.fusesource.jansi.Ansi.ansi;
+
+@Command(scope = "admin", name = "profile-install", description = "Installs the apps and features from a given install profile")
+@Service
+public class ProfileInstallCommand extends AbstractProfileCommand {
+    @Argument(index = 0, name = "profileName", description = "Name of install profile to use", required = true, multiValued = false)
+    String profileName;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProfileInstallCommand.class);
+
+    @Inject
+    protected ConfigurationAdmin configAdmin;
+
+    private FeaturesService featuresService;
+
+    public ProfileInstallCommand() {
+        featuresService = (FeaturesService) getService(FeaturesService.class);
+
+        if (featuresService == null) {
+            LOGGER.debug("Features Service was null");
+        }
+    }
+
+    @Override
+    protected void doExecute(ApplicationService applicationService)
+            throws ApplicationServiceException {
+
+        if (!profileExists(applicationService.getInstallationProfiles(), profileName) && showDevProfiles) {
+            Map<String, List<String>> devProfile = getDevProfile(profileName);
+
+            startApps(applicationService, devProfile.get(DEV_PROFILE_START_APPS));
+
+            if (featuresService != null) {
+                uninstallInstallerModule();
+                installFeatures(devProfile.get(DEV_PROFILE_INSTALL_FEATURES));
+                uninstallFeatures(devProfile.get(DEV_PROFILE_UNINSTALL_FEATURES));
+            }
+
+            stopBundles(devProfile.get(DEV_PROFILE_STOP_BUNDLES));
+
+            console.println(ansi().fgBrightGreen().a("Installation complete.").reset());
```

Comment:
How about moving this console output after the else such that both profiles and dev profiles gets the indication on the console?